### DISCUSSION TITLE
Tailwind crash with watch

### DISCUSSION THREAD

**Discussion by bayaderpack** [tailwindlabs/tailwindcss#14126] [Help]
tailwindcss v3.4.7

Node: v21.5.0

Browser: Chrome

Windows 11

**Describe your issue**

I'm using this command to watch the file changes 

```
tailwindcss -i ./assets/styles.scss -o ./static/styles.css --watch
```
After some time I get this error and tailwind crash 

<--- Last few GCs --->

[19804:00000284BC071B40]  6883361 ms: Mark-sweep 4050.6 (4138.7) -> 4038.8 (4142.7) MB, 3220.5 / 16.2 ms  (average mu = 0.363, current mu = 0.017) allocation failure scavenge might not succeed
[19804:00000284BC071B40]  6888419 ms: Mark-sweep 4054.6 (4142.7) -> 4042.8 (4146.7) MB, 4975.4 / 11.5 ms  (average mu = 0.182, current mu = 0.016) allocation failure scavenge might not succeed

<--- JS stacktrace --->

FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
 1: 00007FF6BA7219EF v8::internal::CodeObjectRegistry::~CodeObjectRegistry+114175
 2: 00007FF6BA6AF836 v8::internal::MicrotaskQueue::GetMicrotasksScopeDepth+65526
 3: 00007FF6BA6B06ED node::OnFatalError+301
 4: 00007FF6BAFE7BFE v8::Isolate::ReportExternalAllocationLimitReached+94
 5: 00007FF6BAFD20FD v8::SharedArrayBuffer::Externalize+781
 6: 00007FF6BAE7545C v8::internal::Heap::EphemeronKeyWriteBarrierFromCode+1468
 7: 00007FF6BAE72574 v8::internal::Heap::CollectGarbage+4244
 9: 00007FF6BAE94A76 v8::internal::Factory::NewFillerObject+214
10: 00007FF6BABC6A75 v8::internal::DateCache::Weekday+1797
11: 00007FF6BB0758F1 v8::internal::SetupIsolateDelegate::SetupHeap+494417
12: 00007FF6BB036A12 v8::internal::SetupIsolateDelegate::SetupHeap+236658
13: 00000284BDEDA4B8

**Comment by RobinMalfait** [comment]
Hey!

Can you share a minimal reproduction repo that we can look at?
